pkg/policyconditions/conditions.go

```diff
@@ -152,8 +152,11 @@ func Update(cli client.Client, policyKey types.NamespacedName) error {
 			if enactmentsCount.Matching() == 0 {
 				message := "Policy does not match any node"
 				setPolicyNotMatching(&policy.Status.Conditions, message)
-			} else if enactmentsCount.Failed() > 0 {
+			} else if enactmentsCount.Failed() > 0 || enactmentsCount.Aborted() > 0 {
 				message := fmt.Sprintf("%d/%d nodes failed to configure", enactmentsCount.Failed(), enactmentsCount.Matching())
+				if enactmentsCount.Aborted() > 0 {
+					message += fmt.Sprintf(", %d nodes aborted configuration", enactmentsCount.Aborted())
```

Comment:
Let's add also the number of matching

Reply:
it is still there, I concatenate it to the original `message`